docs/contributing.md

```diff
@@ -0,0 +1,451 @@
+# Contributing to Python-Markdown
+
+The following is a set of guidelines for contributing to Python-Markdown and its
+extensions, which are hosted in the [Python-Markdown Organization] on GitHub.
+These are mostly guidelines, not rules. Use your best judgment, and feel free to
+propose changes to this document in a pull request.
+
+## Code of Conduct
+
+This project and everyone participating in it is governed by the
+[Python-Markdown Code of Conduct]. By participating, you are expected to uphold
+this code. Please report unacceptable behavior to <markdown@freewisdom.org>.
+
+## Project Organization
+
+The core Python-Markdown code base and any built-in extensions are hosted in the
+[Python-Markdown/markdown] project on GitHub. Other extensions maintained by the
+Python-Markdown project may be hosted as separate repositories in the
+[Python-Markdown Organization] on GitHub and must follow best practices for
+third-party extensions.
+
+The [Python-Markdown/markdown] project is organized as follows:
+
+* Branch `master` should generally be stable and release-ready at all times.
+* Version branches should be used for bug-fixes back-ported to the most recent
+  PATCH release.
+* No other branches should be created. Any other branches which exist are
+  preserved for historical reasons only.
+
+## Issues
+
+Please address issues on the correct channel. Usage questions should be directed
+to the discussion group. Feature requests, bug reports and other issues should
```

Comment:
My understanding is that the old siteforge mailing list (linked in the readme) isn't working any more. The archives still exist, but no new messages are being accepted.

Unless we create some new discussion list somewhere, this comment should be removed. If we do create something, then we should link to it here. Anyone have any suggestions?

Reply:
Some obvious solutions would be Google Groups or Slack. Although, I believe Slack doesn't offer public archives. 

[Groups.io] looks interesting (even has GitHub integration), but I don't know much about it.

[Groups.io]: https://groups.io/

Reply:
Slack, Gitter, Discord, etc are all "chat" solutions, not mailing list solutions. AFAICT, a common feature is that the archives are private to members only. We want something with public archives, which generally means a mailing list. Groups.io appears to be a modern take on mailing lists, but the feature I think we need is the ability to participate outside of an email client. I'm not sure if groups.io supports that. Google Groups certainly does. And its not unusual for people to already have a google account which eliminates the need for people to create another account just to get help.

It is possible to move an old mailman list archive to google groups, but I don't have admin access to the group as it was created before I got involved. I reached put to @yuri when we got notice that SourceForge was shutting the list down, but never heard back.

Another option would be to just use GitHub issues for support questions in addition to bug reports and feature requests. We would need to redefine the `labels` to properly categorize things, but given the amount of questions we get, that is probably sifficient.

Reply:
I would agree that GitHub Issues is probably sufficient.  If we were getting a ton of activity, then I think it would make more sense to have a separate area to chat.